public/static/app/common/PointsDataService.js

```diff
@@ -1,36 +1,70 @@
 /*
  * This service contains functions which can access the points collection.
  */
-mciModule.factory('PointsDataService', function($log, Stitch, STITCH_CONFIG) {
+if (_.isUndefined(_.isSet)) {
+  _.mixin({
+    isSet: function (obj) {
```

Comment:
Is there a less hacky way to check this?

Reply:
I can use isFunction(object) but I'm not convinced it is any better. Why do you consider this hacky?

Reply:
Sorry, I probably should have put this on the line below. Is there a better way to check if an object is a set instead of checking the string representation of it?

Reply:
Nope, the [code](https://github.com/jashkenas/underscore/blob/d5fe0fd4060f13b40608cb9d92eda6d857e8752c/underscore.js#L1326-L1330) is actually from underscore.